drivers/adc/adc_lmp90xxx.c

```diff
@@ -1098,142 +1117,54 @@ static const struct adc_driver_api lmp90xxx_adc_api = {
  * LMP90077: 16 bit, 2 diff/4 se (4 channels), 0 currents
  */
 #if DT_INST_0_TI_LMP90077
```

Comment:
These #if should be something like `#if DT_HAS_COMPAT(ti_lmp90077)`

Reply:
Fixed, thank you.